src/IB/IBFEMethod.cpp

```diff
@@ -884,34 +928,50 @@ IBFEMethod::computeLagrangianForce(const double data_time)
     batch_vec_ghost_update(d_X_half_vecs, INSERT_VALUES, SCATTER_FORWARD);
     for (unsigned part = 0; part < d_num_parts; ++part)
     {
-        if (d_is_stress_normalization_part[part])
+        if (d_part_is_active[part])
```

Comment:
Let's not include this guard --- I think that I would like for all computations to proceed "as usual" for inactivated parts, except that at the end of the day, forces are not spread and velocity is not interpolated.